projects/addon-doc/package.json

```diff
@@ -7,23 +7,24 @@
     "license": "Apache-2.0",
     "repository": "https://github.com/tinkoff/taiga-ui",
     "homepage": "https://github.com/tinkoff/taiga-ui",
+    "dependencies": {
+        "markdown-it": "^12.3.2",
+        "highlight.js": "^11.3.1",
+        "ngx-highlightjs": "^4.1.4",
+        "schematics-utilities": "^1.0.1",
+        "@angular-devkit/schematics": "^9.0.0"
+    },
+    "devDependencies": {
+        "@types/markdown-it": "^12.2.3"
+    },
     "peerDependencies": {
         "@angular/common": ">=9.0.0",
         "@angular/core": ">=9.0.0",
         "@angular/forms": ">=9.0.0",
         "@angular/router": ">=9.0.0",
-        "@angular-devkit/schematics": "^9.0.0",
         "@ng-web-apis/common": "1.12.1",
         "@tinkoff/ng-polymorpheus": "3.1.12",
         "@taiga-ui/cdk": "2.31.0",
```

Comment:
Can you either extend "@ng-web-apis/common" and "@tinkoff/ng-polymorpheus" library version range or move them to "hard dependencies"?

Reply:
I can add like this in peerDependencies

```json
 "@tinkoff/ng-polymorpheus": "^3.1.12",
 "@taiga-ui/cdk": "^2.31.0",
```

but I'm not sure if adding in hard dependencies is the right way

Reply:
Yeah, that solution could work for `@tinkoff/ng-polymorpheus` and `@taiga-ui/cdk`, but it doesn't work for `@ng-web-apis/common` which is already at version 2.

Reply:
Looks good to me 👍